src/sql/workbench/api/node/mainThreadConnectionManagement.ts

```diff
@@ -43,6 +43,19 @@ export class MainThreadConnectionManagement implements MainThreadConnectionManag
 		this._toDispose = dispose(this._toDispose);
 	}
 
+	public $getConnections(activeConnectionsOnly?: boolean): Thenable<azdata.connection.ConnectionProfile[]> {
+		let connections = this._connectionManagementService.getActiveConnections();
+		if (activeConnectionsOnly !== false) {
```

Comment:
activeConnectionsOnly === true... (easier to read)

Reply:
actually its a mistake it should be !== true, this way it will cover the scenario when the parameter is null or undefined. lesson learned: don't send pr when tired. :(

Reply:
Typically for that you'd use the double !, e.g. 

if(!!activeConnectionsOnly) {
    // filter out non-active connections here
}

---
In reply to: [287573972](https://github.com/microsoft/azuredatastudio/pull/5651#discussion_r287573972) [](ancestors = 287573972)